recipes/jupyterlab-daw/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "jupyterlab-daw" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "755d1ddb29b4577b4e5650638563d00125f09f76ad26b1828d0be0af291a9e08" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/j/jupyterlab_daw/jupyterlab_daw-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - hatch-jupyter-builder >=0.5
+    - hatch-nodejs-version
+    - hatchling >=1.4.0
+    - jupyterlab >=3.4.7,<4
+    - pip
+    - python >=3.7,<4.0
+  run:
+    - python >=3.7,<4.0
+
+test:
+  requires:
+    - jupyterlab >=3.4.7,<4
+    - pip
+    - m2-grep  # [win]
+  imports:
+    - jupyterlab_daw
+  commands:
+    - pip check
+    - jupyter labextension list
+    - jupyter labextension list 1>labextensions 2>&1
+    - grep -iE "jupyterlab-daw.*v{{ version.replace(".", "\\.") }}.*enabled.*OK.*jupyterlab_daw" labextensions
+
+about:
+  home: https://github.com/benbovy/jupyterlab-daw
+  license: BSD-3-Clause
+  license_family: BSD
```

Based on what's in the 3rd party directory, it's really:
```suggestion
  license: BSD-3-Clause AND MIT AND 0BSD
```

---

Is it common for the `license` entry to have multiple licenses? Other jupyter extension recipes in conda-forge don't seem to include the license names of the 3rd-party JS dependencies (https://github.com/search?q=org%3Aconda-forge+third-party-licenses.json&type=code&p=1), but I'm happy to apply your suggested change if that's better!

---

Well, I wasn't aware of that file being buried deep in the package files. In an ideal world, they probably should all do this. In this case, at the bare minimum the `license` field should match what's being linked in `license_file`.